tests/integration/targets/mso_schema_template_bd/tasks/main.yml

```diff
@@ -388,7 +411,8 @@
     - nm_add_bd_5.current.l2Stretch == true
     - nm_add_bd_5.current.l2UnknownUnicast == "flood"
     - nm_add_bd_5.current.l3MCast == false
-  when: version.current.version is version('3.1.1g', '<')
+  when: 
+  - version.current.version is version('3.1.1g', '<')
```

Comment:
If single condition, they should be on same line as the when. See below for the others that are similar too.

Reply:
ok